<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadingDeadlineDialog</class>
 <widget class="QDialog" name="DownloadingDeadlineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>176</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Schedule sequential downloading</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Torrent or file name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>of size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>XX GB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>; remaining</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblRemainingSize">
       <property name="text">
        <string>XX GB</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Down&amp;load time</string>
         </property>
         <property name="buddy">
          <cstring>teDeadline</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTimeEdit" name="teDeadline">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Downloading of pieces will be scheduled monotonically with the deadline for the last piece equal to this value.</string>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::HourSection</enum>
         </property>
         <property name="displayFormat">
          <string>HH 'h' mm 'm'  ss 's'</string>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Wi&amp;th speed</string>
         </property>
         <property name="buddy">
          <cstring>sbSpeed</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbSpeed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> kb/s</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="verticalSpacing">
        <number>-1</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Add a dela&amp;y</string>
         </property>
         <property name="buddy">
          <cstring>teDealy</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTimeEdit" name="teDealy">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Downlaoding deadlines will be delayed for this amount of time.</string>
         </property>
         <property name="displayFormat">
          <string>HH 'h' mm 'm'  ss 's'</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="chbxUncompletedOnly">
       <property name="text">
        <string>Unfinished only</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sbSpeed</tabstop>
  <tabstop>teDeadline</tabstop>
  <tabstop>teDealy</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DownloadingDeadlineDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DownloadingDeadlineDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
